src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/ModelComputationsHelper.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file ModelComputationsHelper.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_FLOATING_BASE_ESTIMATORS_MODEL_COMPUTATIONS_HELPER_H
+#define BIPEDAL_LOCOMOTION_FLOATING_BASE_ESTIMATORS_MODEL_COMPUTATIONS_HELPER_H
+
+// std
+#include <memory>
+
+// iDynTree
+#include <iDynTree/KinDynComputations.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+
+/**
+ * KinDynComputationsDescriptor wraps a pointer iDynTree KinDynComputations object.
+ */
+struct KinDynComputationsDescriptor
+{
+    std::shared_ptr<iDynTree::KinDynComputations> kindyn{nullptr}; /**< Pointer associated to the KinDynComputations. */
+
+    /**
+     * Constructor.
+     */
+    KinDynComputationsDescriptor(std::shared_ptr<iDynTree::KinDynComputations> kindyn);
+
+    /**
+     * Constructor.
+     */
+    KinDynComputationsDescriptor();
+
+    /**
+     * Check if the poly driver descriptor is valid.
+     * @return True if the polydriver is valid, false otherwise.
```

Comment:
ctrl+c ctrl+v fails: episode Inf

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/216/commits/2458b2f7555d7791ce8ceced67626d8a45c031cf